### ISSUE TITLE
validate does not check all the files at once

### ISSUE THREAD

**Issue by victornoel** [revelc/formatter-maven-plugin#270]
When I run the `validate` goal, it fails at the first problematic file instead of showing me all the files that fail.

If the objective is to use it in the CI for example, this is very limiting to only show one error…

**Comment by hazendaz** [comment]
Not really.  It simply means formatting was not run.  So when CI fails, developer should run a proper build and commit the result which may be many files.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by victornoel** [comment]
I understand that, but it would be nicer if we could get all at once: even locally, it is tedious to run `validate` one file per one file to correct each of them.

It is not always possible to use eclipse's formatting on a whole project, for example if one configure some exclusions from the validation.
